library/src/main/res/values-v21/styles.xml

```diff
@@ -1,5 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+   <style name="Chucker.Theme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
Can I ask you to refactor to have a:

`<style name="Chucker.BaseTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">`

inside the `res/values/styles.xml` with all the common theme attributes?

We can then have the `Chucker.Theme` defined in both `values` and `values-v21` so we don't need to copy over the theme twice.

Reply:
Sure. I just thought that we might bump minSDK to 21 pretty soon due to Retrofit updates, etc.

Reply:
Yeah true, but we don't have a PR pending for that (plus would be a major version bump). I think it's better if we keep the theme clean. Moreover having this setup with a `BaseTheme` could be handy for future similar scenarios